content/en/docs/components/pipelines/v1/overview/multi-user.md

```diff
@@ -1,175 +1,178 @@
+
+# create a pipeline run
+client.run_pipeline(
+    experiment_id="<YOUR_EXPERIMENT_ID>",  # the experiment determines the namespace
+    job_name="<YOUR_RUN_NAME>",
+    pipeline_id="<YOUR_PIPELINE_ID>"  # the pipeline definition to run
+)
+print(client.list_runs(experiment_id="<YOUR_EXPERIMENT_ID>"))
+print(client.list_runs(namespace=user_namespace))
+```
 
-Detailed documentation for the Kubeflow Pipelines SDK can be found in the
-[Kubeflow Pipelines SDK Reference](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.client.html).
+{{% alert title="Tip" color="info" %}}
+* To set a default namespace for Pipelines SDK commands, use the [`kfp.Client().set_user_namespace()`](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.client.html#kfp.Client.set_user_namespace) method, 
+  this method stores your user namespace in a configuration file at `$HOME/.config/kfp/context.json`.
+* Detailed documentation for `kfp.Client()` can be found in the [Kubeflow Pipelines SDK Reference](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.client.html).
+{{% /alert %}}
 
-### When using REST API or generated Python API client
+## When using the REST API
 
-Similarly, when calling [REST API endpoints](/docs/components/pipelines/reference/api/kubeflow-pipeline-api-spec/)
-or using [the generated Python API client](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.server_api.html),
-namespace argument is required for experiment APIs. Note that namespace is
-referred to using a resource reference. The resource reference **type** is
-`NAMESPACE` and resource reference **key id** is the namespace name.
+When calling the [Kubeflow Pipelines REST API](/docs/components/pipelines/reference/api/kubeflow-pipeline-api-spec/), a namespace argument is required for experiment APIs.
+<br>
+The namespace is specified by a "resource reference" with `type` of `NAMESPACE` and `key.id` equal to the namespace name.
 
-The following example demonstrates how to use [the generated Python API client (kf-server-api)](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.server_api.html) in a multi-user environment.
+The following code uses the [generated python API client](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.server_api.html) to create an experiment and pipeline run.
 
 ```python
-from kfp_server_api import ApiRun, ApiPipelineSpec, \
-    ApiExperiment, ApiResourceType, ApiRelationship, \
-    ApiResourceReference, ApiResourceKey
-# or you can also do the following instead
-# from kfp_server_api import *
-
-experiment=client.experiments.create_experiment(body=ApiExperiment(
-    name='test-experiment-1234',
-    resource_references=[ApiResourceReference(
-        key=ApiResourceKey(
-            id='<namespace>', # Replace with your own namespace.
-            type=ApiResourceType.NAMESPACE,
-        ),
-        relationship=ApiRelationship.OWNER,
-    )],
-))
+import kfp
+from kfp_server_api import *
```

Comment:
nit: Why `import *`? This is generally a discouraged import pattern.

Reply:
@connor-mccarthy I agree, I just left it the same as it was, feel free to raise a quick PR to fix it?